karma.conf.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (config) {
       environment: 'dev'
     },
     reporters: ['mocha'],
+    reportSlowerThan: 100,
```

Comment:
We're going to get a yellow warning next to each test that exceeds 100ms in execution. I know 100ms for a unit test is a lot but I'd like to filter out the longest tests, which are a few seconds long. Having a treshold of 100ms will help to find them more easily in the console log.

Reply:
👍